### DISCUSSION TITLE
I want to take part in new project as a backend developer.

### DISCUSSION THREAD

**Discussion by Binary-Ninja-007** [community/community#64231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

🤞 Hi, Everybody! 🤞

I am Jack London from Singapore.  As a Full-Stack Developer, I am good at using several programming languages such as JavaScript, TypeScript, Java, PHP, Django and C# ,etc. I have rich experience in developing several kinds of projects.
Now I have been seeking a new project that I can join and collaborate.
My broad experience has Shopify development, WordPress Plugin development and Map integration with React.js.
I am always using modern framework to develop such as .Net Core, Angular, React.js, Vue.js, Django, Spring and Laravel.
My algorithm for development is very clear and strong, and so I have got positive feedback about my skill from my past clients.
If you interested in me, Let's work together.

**Comment by Jaber-Saed** [comment]
Hello Jack London,

It's great to hear about your diverse skill set and experience as a Full-Stack Developer. Your proficiency in a wide range of programming languages and frameworks, along with your past project involvement, demonstrates a strong foundation for software development.

If you're seeking new projects to collaborate on, here are a few steps you might consider:

1. **Portfolio Showcase**: Make sure you have a portfolio that showcases your past projects and technologies you've worked with. This can help potential collaborators understand your capabilities.

2. **GitHub Profile**: Having an active and well-maintained GitHub profile can showcase your coding style, contributions to open source projects, and your overall engagement with the developer community.

3. **Networking**: Connect with other developers and professionals on platforms like LinkedIn, Twitter, and relevant online communities. Networking can open up opportunities for collaborations and partnerships.

4. **Freelance Platforms**: Consider platforms like Upwork, Freelancer, or Toptal to find freelance projects that match your skill set and interests.

5. **Open Source Contributions**: Participating in open source projects can not only help you improve your skills but also connect you with like-minded developers and potential collaborators.

6. **Job Boards**: Check job boards, both local and remote, for opportunities that align with your skills and interests.

7. **Personal Projects**: If you have ideas of your own, you can start personal projects or side projects to showcase your creativity and technical abilities.

8. **Industry-Specific Platforms**: Depending on the industries you're interested in, there might be specialized platforms or forums where you can connect with potential collaborators.

9. **Reach Out**: Don't hesitate to reach out to companies or individuals whose projects align with your expertise. Sending a thoughtful introduction email or message can sometimes lead to collaborations.

10. **Local Meetups and Events**: Participate in local tech meetups, conferences, and events. These can be great opportunities to meet fellow developers and potential collaborators face-to-face.

Remember that building relationships and trust in the tech community takes time, so be patient and persistent in your search for new collaborations. Your diverse skill set and experience should make you an attractive candidate for many projects. Best of luck on your journey!

Warm regards,
Jaber

**Reply by Binary-Ninja-007** [reply]
Thanks, Jaber.

**Comment by rustdesk** [comment]
Wondering if you have interest to learn Rust, I am from Singapore (Yishun), currently I am maintaining https://github.com/rustdesk/rustdesk.

**Reply by Binary-Ninja-007** [reply]
Hi, I am interested in Rust. Could you explain more about rust?

**Reply by Binary-Ninja-007** [reply]
Because I have rich experience in development, I am sure I can learn Rust faster than others.
If you need my help to maintain your site, I can help you.

**Reply by rustdesk** [reply]
My project is not a website, though it does have web version.

**Reply by Binary-Ninja-007** [reply]
What kind of programming language did you use for your project?

**Reply by Binary-Ninja-007** [reply]
Dear, I think Rust is similar with C++. C++ is OOP. Could u tell me the difference between C++ and Rust?

**Reply by rustdesk** [reply]
ChatGPT can help you.

**Comment by Binary-Ninja-007** [comment]
Thank you for your kindness.
Don't you need my help for development of you project, (I mean web version)?

**Comment by sanyaoyega** [comment]
Good day Chief,
   I currently work as a junior recruiter for a web3 recruitment platform and we have three positions that you might want to check out. 
   Two of these positions require developers based in Singapore. I advise that you play to your strengths and also state that you are based in Singapore. These startups really need to hire fast,so you have a great chance of getting employed. Good luck!!

1. Technical lead: https://intro.ac/V5eNmLj

2. Senior Backend Developer
https://intro.ac/uIfaIJF

3. Fullstack Developer:
https://intro.ac/en3H9Re

**Comment by Wahaj95** [comment]
Hi,
This is syed wahaj from TechKluster, They are hiring developers for their domain, You can apply and learn a lot from there. If you are interested, you can simply apply on [app.techklutser.com](https://app.techkluster.com/)
Plus if anyone want to hire resource like [hire java developer](https://techkluster.com/java/) they can go for the mentioned link as well.

**Comment by nouraalali** [comment]
I am hiring as well- AWS and .Net expierenced developers

noura@bamboo-card.com

If you are open to being in office in Dubai- even better.
